src/main/java/org/folio/mr/listener/kafka/KafkaEventListener.java

```diff
@@ -0,0 +1,83 @@
+package org.folio.mr.listener.kafka;
+
+
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+
+import org.folio.mr.domain.dto.Request;
+import org.folio.mr.exception.KafkaEventDeserializationException;
+import org.folio.mr.service.KafkaEventHandler;
+import org.folio.mr.service.impl.RequestEventHandler;
+import org.folio.mr.support.KafkaEvent;
+import org.folio.spring.integration.XOkapiHeaders;
+import org.folio.spring.service.SystemUserScopedExecutionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import lombok.extern.log4j.Log4j2;
+
+@Component
+@Log4j2
+public class KafkaEventListener {
+  private static final ObjectMapper objectMapper = new ObjectMapper();
+  private final RequestEventHandler requestEventHandler;
+  private final SystemUserScopedExecutionService systemUserScopedExecutionService;
+
+  public KafkaEventListener(@Autowired RequestEventHandler requestEventHandler,
+                            @Autowired SystemUserScopedExecutionService systemUserScopedExecutionService) {
+
+    this.requestEventHandler = requestEventHandler;
+    this.systemUserScopedExecutionService = systemUserScopedExecutionService;
+  }
+
+  @KafkaListener(
+    topicPattern = "${folio.environment}\\.\\w+\\.circulation\\.request",
+    groupId = "${spring.kafka.consumer.group-id}"
+  )
+  public void handleRequestEvent(String eventString, MessageHeaders messageHeaders) {
+    log.debug("handleRequestEvent:: event: {}", () -> eventString);
+    KafkaEvent<Request> event = deserialize(eventString, messageHeaders, Request.class);
+    log.info("handleRequestEvent:: event received: {}", event::getId);
```

Comment:
This way of logging is deprecated. Will `event.getId()` make any performance downgrade?